### DISCUSSION TITLE
Clipbard text share

### DISCUSSION THREAD

**Discussion by HubKing** [slowscript/warpinator-windows#13] [Ideas]
A lot of times, I need to share a little bit of string (such as a URL, a password, an e-mail address, etc) between two devices. As far as I know, Apple has some sort of clipboard sharing feature between mac and iOS devices to facilitate this. Can Warpinator add clipboard text sharing? I guess it needs both clients need to support this. If the other client support this, the other client adds the received string to the clipboard (so that the user can press Ctrl+V to paste it somewhere), and if the other client does not support clipboard, then it could be sent as a plain text file.

I tried selecting some text on Android, and then chose "Share", and then selected Warpinator, but it said "Nothing to share".

**Comment by slowscript** [comment]
I plan to implement some form of text sharing in the future, although I'll probably first do it in the Linux version, offer the feature to the original authors and get their feedback.
It won't be a continuous clipboard share though. That would require some way to configure which devices it should be shared with and I don't think pasting the clipboard content into Warpinator manually is that difficult.
